src/config/site.config.js

```diff
@@ -0,0 +1,75 @@
+const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
+const navigationConfig = require('./navigation.config');
+
+module.exports = {
+  /* The path to the navigation configuration.  */
+  navConfig: navigationConfig,
+
+  // /** The array of pathes and/or regex patterns to the page and tests examples
```

Comment:
See long comment above. I want to include these keys once we know how to generate the component config and consumer it dynamically.